roles/flent/templates/configmap_script.yml.j2

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
Where is this configmap being used?

Reply:
Good catch. It's no longer being used. I'll remove it.